index.html

```diff
@@ -0,0 +1,352 @@
+            <img
+              class="image-fluid"
+              src="images/works/image_3.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h3 class="title">Tonic</h3>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="btn-container-flex">
+              <li class="btn">
+                <button class="btn-sm" type="button">html</button>
+              </li>
+              <li class="btn">
+                <button class="btn-md" type="button">css</button>
+              </li>
+              <li class="btn">
+                <button class="btn-lg" type="button">javaScript</button>
+              </li>
+            </ul>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="card-layout"> 
+          <a href="#">
+            <img
+              class="image-fluid"
+              src="images/works/image_4.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h3 class="title">Multi-Post <br>
+              Stories</h3>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="btn-container-flex">
+              <li class="btn">
+                <button class="btn-sm" type="button">html</button>
+              </li>
+              <li class="btn">
+                <button class="btn-md" type="button">css</button>
+              </li>
+              <li class="btn">
+                <button class="btn-lg" type="button">javaScript</button>
+              </li>
+            </ul>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="about">
+      <h2 class="about-title">About<br />Myself<br /></h2>
+      <p class="paragraph">
+        Hello I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me..
+      </p>
```

Comment:
- [ ] Great job but I think here just you need to fix the alignment to be in the same line to match the Figma as much as possible so kindly fix it 💯

- [ ] I think it would be great if you add some font weight on the `Get my resume` button a to make it the same as figma.
- your design
![image](https://user-images.githubusercontent.com/58553711/185278704-5b9819ba-fd02-4235-b806-6dfa23eac2d5.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/185278764-a1bea19c-cc5d-4c8b-8539-c1b89f58638b.png)